lib/ibmdb.js

```diff
@@ -757,18 +765,11 @@ IBMDB.prototype.buildIndexes = function(model) {
                       ' (\"' + columns + '\");\n');
   }
 
-  // Define index for each of the properties
-  // for (var p in definition.properties) {
-  //   var propIndex = this.buildIndex(model, p);
-  //   if (propIndex) {
-  //     indexClauses.push(propIndex);
-  //   }
-  // }
-
   return indexClauses;
 };
 
 IBMDB.prototype.buildColumnDefinition = function(model, prop) {
+  debug('IBMDB.prototype.buildColumnDefinition');
```

Comment:
Fixing now.  On your other comments, I haven't back fit to 2.x yet so will need some assistance with that.